Signal/src/ViewControllers/InboxTableViewCell.m

```diff
@@ -184,68 +272,11 @@ - (NSAttributedString *)dateAttributedString:(NSDate *)date {
     return attributedString;
 }
 
-- (void)setUnreadMsgCount:(NSUInteger)unreadMessages {
```

Comment:
We were using a pretty elaborate method to render the unread badges in the home view cells.  I replaced this with something much simpler (see above).